source/reference/minio-server/settings/minio-client.rst

```diff
@@ -0,0 +1,83 @@
+.. _minio-server-envvar-mc:
+
+=====================
+MinIO Client Settings
```

Comment:
@djwfyi almost - this has nothing to do with just interacting with the server.

You can use these envvars for mc only configs, including just creating aliass for non-MinIO deployments.

IMO move this under `minio/mc.rst` as a child page as there are other envvars for mc I think we do not cover.

Reply:
Moving.